Instruments/bridge12.py

```diff
@@ -324,19 +298,34 @@ def set_freq(self,Hz):
                            "frequency to change -- result is %d setting is %d"%(result,setting))
     def get_freq(self):
         return self.freq_int()*1e3
-    def freq_int(self):
-        "return the frequency, in kHz (since it's set as an integer kHz)"
-        self.write(b'freq?\r')
+    def robust_int_response(self,cmd,numtries=10):
         for j in range(10):
```

Comment:
I didn't add the additional reset_input buffer because we already do that and if the conditionals below in the loop I think are useful if there IS an error, but everything else is added